### PR TITLE
Fix place display [skip deploy]

### PR DESCRIPTION
**Pull request by mgwalker**
Sends back useful user information from the API when logging in or checking your current authentication status.  Captures that user information in the app and stores it in the user application state.  Then we can do cool stuff with it.

### This pull request changes...
- the state name and graphic displayed in the sidebar will match the state associated with your user account

### This pull request is ready to merge when...
- [x] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- [x] The change has been documented
  - [x] Associated OpenAPI documentation has been updated

### CODE REVIEW

File: api/db/user.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ module.exports = (zxcvbn = defaultZxcvbn, bcrypt = defaultBcrypt) => ({
 
     async apds() {
       logger.silly('getting user apds');
-      if (!this.relations.state || !this.relations.state.apds) {
+      if (!this.relations.state || !this.relations.state.relations.apds) {
```
Comment (mgwalker) [inline api/db/user.js]: This has always been wrong.  Thankfully we never used it before.  😛

File: web/src/reducers/user.js [hunk 2]
```diff
@@ -30,6 +37,8 @@ const user = (state = initialState, action) => {
       };
     case GET_USER_FAILURE:
       return { ...state, fetching: false, error: action.error };
+    case LOGOUT_SUCCESS:
+      return initialState;
```
Comment (mgwalker) [inline web/src/reducers/user.js]: It occurs to me that we should probably reset the state globally on logout.

**Comment by mgwalker** [comment]
No PR deployment for this one because it requires API changes.

File: web/src/reducers/user.js (hunk 2 above)
Comment (brendansudol) [reply web/src/reducers/user.js]: yes we should

**Pull request merged**